### PR TITLE
11 4j integrate mapbox in app

### PR DESCRIPTION
**Pull request by Neyross**

### CODE REVIEW

File: app/src/androidTest/java/com/bouboulecorp/thurii/Map/MapFragmentTest.kt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package com.bouboulecorp.thurii.Map
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/Map/MapFragmentTest.kt:1]: 🚫 **[ktlint]** <final-newline> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>File must end with a newline (\n)

File: app/src/androidTest/java/com/bouboulecorp/thurii/Map/MapFragmentTest.kt [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import android.view.View
+import androidx.fragment.app.testing.launchFragmentInContainer
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/Map/MapFragmentTest.kt:7]: 🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)

File: app/src/androidTest/java/com/bouboulecorp/thurii/Map/MapFragmentTest.kt [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner
+import com.bouboulecorp.thurii.HomePage.HomePageFragments.MapFragment
+import com.bouboulecorp.thurii.R
+import org.hamcrest.CoreMatchers.allOf
+import org.hamcrest.CoreMatchers.containsString
+import org.hamcrest.Matcher
+import org.junit.Test
+import org.junit.runner.RunWith
+
+
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/Map/MapFragmentTest.kt:17]: 🚫 **[ktlint]** <no-consecutive-blank-lines> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Needless blank line(s)

**Review by larroyo3 — APPROVED** [review]

File: app/src/androidTest/java/com/bouboulecorp/thurii/Map/MapFragmentTest.kt [hunk 4]
```diff
@@ -0,0 +1,29 @@
+package com.bouboulecorp.thurii.Map
+
+import android.view.View
```
Comment (Neyross) [inline app/src/androidTest/java/com/bouboulecorp/thurii/Map/MapFragmentTest.kt:3]: 🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between

**Pull request merged**